core/Client.php

```diff
@@ -365,7 +365,7 @@ private function parseResponse($response)
      */
     private function parseExceptionResponse($e)
     {
-        $body = $e->getResponse()->getBody(true);
+        $body = $e->getResponse()->getBody(true)->getContents();
```

Comment:
This is just a PSR7 stream, so we use `getContents()` to return just the body string rather than all of the methods and magic attached that isn't being used here.